209 manage teacher user admin fix

barumski opened the pull request:
FIxes for issues #209

-Html file moved to templates.
-Stylesheet link added.
-Icon directions added.
-Removed the duplicate icons and images.
-Border weight in header reduced.
-Found and fixed a little typo.
-Updated footer.

I have not done any styling for the changes, which might be  @needed.

BiankaSzabo committed:
Add HTML-CSS file

BiankaSzabo committed:
Merge branch 'develop' into 209-css-html-manage-teacher-user-admin

barumski committed:
fixes added

barumski committed:
Merge branch 'develop' into 209-manage-teacher-user-admin-fix

kristinatonnessen reviewed (changes requested):
![Skjermbilde 2025-01-15 kl  22 31 33](https://github.com/user-attachments/assets/04290a3e-8bcd-4ac6-b0a4-a06c7d14b916)
assets/51c0e615-3e29-4b63-9fa5-3288deed5be2)
good work 😃 we still need to change/remove the sidebar as its not compatible with the design

Janicke92 reviewed (dismissed):
Code looks good!

barumski committed:
removed sidebar html

barumski commented:
> ![Skjermbilde 2025-01-15 kl 22 31 33](https://private-user-images.githubusercontent.com/142672432/403592171-04290a3e-8bcd-4ac6-b0a4-a06c7d14b916.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc0MTQ0ODEsIm5iZiI6MTczNzQxNDE4MSwicGF0aCI6Ii8xNDI2NzI0MzIvNDAzNTkyMTcxLTA0MjkwYTNlLThiY2QtNGFjNi1iMGE0LWEwNmM3ZDE0YjkxNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEyMFQyMzAzMDFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNThiMGQ4MGI5MDYzMGQ1OGJiNzJhMzcxYjUxMGVhOWMzNDYxNGU5YjI5MDlmZDA0Y2IwYjNlNjUzOGZhYmEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.-HvFFKzw4wVSwHDIUAwRIKGT9CWfW1PjuuBKBNVm6q8) assets/51c0e615-3e29-4b63-9fa5-3288deed5be2) good work 😃 we still need to change/remove the sidebar as its not compatible with the design

Removed the sidebar and fixed the css to make it the page fit on screen. :)

kristinatonnessen reviewed (approved):
looks good 😄 When header and footer is ready we need to add the updated code, but that is a separate issue
